tests/integration_tests.rs

```diff
@@ -57,9 +60,64 @@ async fn test_lndk_forwards_onion_message() {
     // Notice that ldk1 and ldk2 are not connected directly to each other.
     let (pubkey, addr) = ldk1.get_node_info();
     let (pubkey_2, addr_2) = ldk2.get_node_info();
+
     lnd.connect_to_peer(pubkey, addr).await;
     lnd.connect_to_peer(pubkey_2, addr_2).await;
     let lnd_info = lnd.get_info().await;
+    let lnd_pubkey = PublicKey::from_str(&lnd_info.identity_pubkey).unwrap();
```

Comment:
So... before LDK 20 this integration test worked ok without channels set up. But now, it doesn't. IIUC LDK 20 requires a node to be in the network graph so it can find a way to forward a message, meaning channels need to be set up rather than just connections to peers. Planning to look into this further and ask the LDK team about it.

Reply:
Fixed this with a small PR to rust-lightning to export a lower level version of "send_onion_message" in ldk-sample so we can set the path directly (like we did before) and get around the channels-required issue.